Userland/Applications/SoundPlayer/main.cpp

```diff
@@ -124,8 +124,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(visualization_menu->try_add_action(samples));
     visualization_actions.add_action(samples);
 
-    auto none = GUI::Action::create_checkable("&None", [&](auto&) {
-        static_cast<SoundPlayerWidgetAdvancedView*>(player)->set_visualization<NoVisualizationWidget>();
+    auto none = GUI::Action::create_checkable("&Album Cover", [&](auto&) {
```

Comment:
We should probably rename the action variable as well. 
```suggestion
    auto album_cover_vizualizaton = GUI::Action::create_checkable("&Album Cover", [&](auto&) {
```

Reply:
Totally missed that one!